src/components/Personalization/createOnClickHandler.js

```diff
@@ -12,7 +12,10 @@ governing permissions and limitations under the License.
 
 export default ({ mergeMeta, collectClicks, clickStorage }) => {
   return ({ event, clickedElement }) => {
-    const merger = meta => mergeMeta(event, meta);
+    const merger = meta => {
+      event.mergeXdm({ eventType: "click" });
```

Comment:
{eventType: "web.webinteraction.linkClicks"} is what we have used for the ActivityMap stuff.

Reply:
True, but this is not really a `linkClick` it can be anything, so ideally XDM should allow us to send either freeform event type or we should add other types along `linkClick`. This should apply to mobile as well.